nixos/modules/services/web-servers/nginx/default.nix

```diff
@@ -189,20 +189,15 @@ let
 
       ${optionalString (cfg.recommendedGzipSettings) ''
         gzip on;
-        gzip_proxied any;
-        gzip_comp_level 5;
-        gzip_types
-          application/atom+xml
-          application/javascript
-          application/json
-          application/xml
-          application/xml+rss
-          image/svg+xml
-          text/css
-          text/javascript
-          text/plain
-          text/xml;
+        gzip_static on;
         gzip_vary on;
+        gzip_disable "MSIE [1-6]\.";
+        gzip_http_version 1.1;
```

Comment:
What is this line doing and how does it differ from the default?

Reply:
It is the default https://nginx.org/en/docs/http/ngx_http_gzip_module.html#gzip_http_version

Reply:
How about dropping default values and just add a link to the official documentation. I like to have less code rather than more because than I have to do the opposite and look up how our nixos configuration differs from a vanilla nginx.

Reply:
The default values have been removed.